src/pageObject.js

```diff
@@ -2,15 +1,41 @@
+"use strict";
 
 class Header {
-
-};
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
+    this.url += slug;
+  }
 
-};
+  get commentButton() {
+    return "[Publish comment] button";
+  }
+
+  clickOnCommentButton() {
+    return `Click on the ${this.commentButton}`;
+  }
+
+  assertPageOpened() {
+    return `The ${this.url} is opened`;
```

Comment:
The `assertPageOpened` method currently returns a string. In a real-world scenario, this method would probably perform some assertion to verify that the expected page has been opened. If this is a placeholder for future implementation, consider adding a comment to clarify this.